src/System.Private.CoreLib/shared/System/Text/StringBuilder.cs

```diff
@@ -1211,6 +1211,15 @@ public StringBuilder Append(ReadOnlySpan<char> value)
             return this;
         }
 
+        public StringBuilder Append(ReadOnlyMemory<char> value)
+        {
+            if (value.Length > 0)
+            {
+                return Append(value.Span);               
+            }
```

Comment:
There's no need to check .Length. The whole implementation can just be `=> Append(value.Span);`

Reply:
done.